src/popup/utils/notifications.ts

```diff
@@ -0,0 +1,64 @@
+import { fetchJson, postJson } from './helper';
+import {
+  IChainNames, INetwork, INotification, NotificationStatus,
+} from '../../types';
+import { AGGREGATOR_URL } from './constants';
+
+// TODO - type sdk
+export const getAllNotifications = async (sdk: any, activeNetwork: INetwork, address: string) => {
```

Comment:
Word `get` in programming means it's a getter - synchronous thing. For the sake of connecting with the API I suggest to use `fetch...` to indicate what actually this method do.